UniExtract.au3

```diff
@@ -71,7 +71,7 @@ Const $bindir = @ScriptDir & "\bin\"
 Const $langdir = @ScriptDir & "\lang\"
 Const $defdir = @ScriptDir & "\def\"
 Const $updater = @ScriptDir & '\UniExtractUpdater.exe'
-Const $unicodepattern = "(?i)(?m)^[\w\Q @!�$%&/\()=?,.-:+~'��{[]}*#߰^��������������������\E]+$"
+Const $unicodepattern = "(?i)(?m)^[\w\Q @!§$%&/\()=?,.-:+~'²³{[]}*#ß°^âëöäüîêôûïáéíóúàèìòù\E]+$"
```

Comment:
Because file here is ANSI codified, while you committed in UTF-8

Final displayed characters in modem programs are the same then, but actual bytes "producing" them aren't.